model/metrics/v8js-metrics.yaml

```diff
@@ -0,0 +1,92 @@
+groups:
+  - id: metric.veightjs.gc.duration
+    type: metric
+    metric_name: v8js.gc.duration
+    brief: "Garbage collection duration."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    prefix: v8js.gc
+    attributes:
+      - ref: v8js.gc.type
+        requirement_level: recommended
+
+  - id: metric.veightjs.memory.total
+    type: metric
+    metric_name: v8js.memory.total
+    brief: "Total memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.veightjs.memory.used
+    type: metric
+    metric_name: v8js.memory.used
+    brief: "Used memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.veightjs.heap.total
+    type: metric
+    metric_name: v8js.heap.total
+    brief: "Total heap memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.veightjs.heap.used
+    type: metric
+    metric_name: v8js.heap.used
+    brief: "Used heap memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+
+  - id: metric.veightjs.heap.space_size
+    type: metric
+    metric_name: v8js.heap.space_size
+    brief: "Heap space size."
```

Comment:
Same with this one. It would be better to describe a bit more what "heap space size" here means.